contrib/utilities/indent

```diff
@@ -21,6 +21,21 @@
 ##
 ## ---------------------------------------------------------------------
 
+find_astyle() {
+  # Split $PATH variable into an array
+  setopt shwordsplit 2>/dev/null # zsh compatibility
```

Comment:
You don't actually need this -- the first line of the script determines that we run this file with bash, whether or not that is the default on a system or not.

Reply:
True. There's no reason that macOS would stop shipping bash alongside zsh. If zsh isn't important, we can toss the last two commits in this PR, since they both make no difference in bash. Should I do that?

Reply:
Yes. The script is run in bash, and we have no intention of changing that.